internal/pkg/db/mongo/metadata.go

```diff
@@ -1034,13 +1031,12 @@ func (m *MongoClient) AddCommand(c *contract.Command) error {
 	s := m.session.Copy()
 	defer s.Close()
 
+	c.Id = ""
+	c.Created = 0
```

Comment:
Is there a specific use case you saw that requires these reset lines? Also, c parameter should not be a pointer. For example:

https://github.com/edgexfoundry/edgex-go/blob/29a4bebcaf459d7aad88a7f62a5784fcebb874d5/internal/pkg/db/mongo/metadata.go#L643

Reply:
Thinking was that contract.Command passed into AddCommand shouldn't be able to dictate its Id, Uuid, or Created fields.

Removed assignments.  Also corrected the AddCommand function signature.